automated_quality_control.py

```diff
@@ -0,0 +1,181 @@
+import os
+import random
+import sys
+from datetime import datetime
+
+import requests
+from retry import retry
+
+API_TOKEN = os.environ.get("SCREENLY_API_TOKEN")
+REQUEST_HEADERS = {
+    "Authorization": f"Token {API_TOKEN}",
+    "Content-Type": "application/json",
+}
+
+
+def get_ten_random_assets():
+    """
+    Return 10 random assets in the account.
+    """
+
+    response = requests.get(
+        "https://api.screenlyapp.com/api/v3/assets/", headers=REQUEST_HEADERS
+    )
+
+    if not response.ok:
```

Comment:
```suggestion
    response.raise_for_status()
```
I guess it will be better and will provide more information

Reply:
and the same for other places